.gitignore

```diff
@@ -79,9 +79,6 @@ SuiteCRM.kdev4
 vendor/
 public/
 
-#Ingore composer files
-composer.lock
```

Comment:
Do we need composer.lock for hotfix?

Reply:
Not for me, there is a workaround: `rm composer.lock` if I don't want to keep it.